docs/scripting/functions/PlayerGangZoneHide.md

```diff
@@ -0,0 +1,56 @@
+---
+title: PlayerGangZoneHide
+description: Hide player gangzone
+tags: ["player", "gangzone", "playergangzone"]
+---
+
+## Description
+
+Hide player gangzone.
+
+| Name        | Description                                                      |
+| ----------- | ---------------------------------------------------------------- |
+| playerid    | The ID of the player to whom player gangzone is bound.           |
+| zoneid      | The ID of the player gangzone for hide.                          |
+
+## Returns
+
+1: The function executed successfully. Success is reported even if the player gangzone was hide to begin with.
+
+0: The function failed to execute. The gangzone specified does not exist.
+
+## Examples
+
+```c
+// This variable is used to store the id of the gangzone
+// so that we can use it throught the script
+new gGangZoneID[MAX_PLAYERS] = {INVALID_GANG_ZONE, ...};
+
+public OnPlayerConnect(playerid)
+{
+    // Create the gangzone
+    gGangZoneID[playerid] = CreatePlayerGangZone(playerid, 2236.1475, 2424.7266, 2319.1636, 2502.4348);
+}
+
+public OnPlayerDeath(playerid, killerid, reason)
+{
+    if(gGangZoneID[playerid] != INVALID_GANG_ZONE)
+        PlayerGangZoneHide(playerid, gGangZoneID[playerid]);
```

Comment:
```suggestion
```

Reply:
I think it’s worth using `IsValidPlayerGangZone `here too instead of  `gGangZoneID[playerid] != INVALID_GANG_ZONE`.

Reply:
Sorry, I’m still not very familiar with this Github interface.